### PR TITLE
Assembly Line Scheduling : PHP and Python added

### PR DESCRIPTION
**Pull request by Karna98**
Fixes 774 (Already Referred by [PR-1254](https://github.com/jainaman224/Algo_Ds_Notes/pull/1254))

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Implementation of Assembly Line Scheduling

#### Languages Used:
- Php
- Python

#### Files Added:
- Assembly_Line_Scheduling/Assembly_Line_Scheduling.php
- Assembly_Line_Scheduling/Assembly_Line_Scheduling.py

Thanks!

### CODE REVIEW

**Comment by Karna98** [comment]
> Please add spaces around operators and after statements like for, if throughout the file.

Please check the latest commit. Required changes are committed.

File: Assembly_Line_Scheduling/Assembly_Line_Scheduling.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+                         /    *  /    *
+        10(In2)     --> 50 --> 70 --> 50 --> 30(Out2)
+    '''
+
+    # lineA, lineB time required at each node on assembly line                                                 Here, lineA = line[0], lineB = line[1] 
+    # aToB, bToA time required at change assembly line from Ai to Bj or Bi to Aj where j = i+1                 Here, aToB = transactionTime[0], bToA = transactionTime[1]
+    # inA, inB Time required to enter Assembly line                                                            Here, inA = inTime[0], inB = inTime[1]  
+    # outA, outB Time required to exit from Assembly line                                                      Here, outA = outTime[0], outB = outTime[1]
+
+    nodes = int(input("Enter the no. of nodes on Assembly line : "));
```
Comment (MastersAbh) [inline Assembly_Line_Scheduling/Assembly_Line_Scheduling.py]: please remove semicolon

File: Assembly_Line_Scheduling/Assembly_Line_Scheduling.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    
+    print("\n-- Transaction Time --")
+    for lineN in range(ASSEMBLY_LINES):
+        tmp = []
+        for nThNode in range(nodes):
+            if (nThNode==0):
+                tmp.append(0)
+                print()
+            else:
+                if (lineN == 0):
```
Comment (MastersAbh) [inline Assembly_Line_Scheduling/Assembly_Line_Scheduling.py]: space around ==

File: Assembly_Line_Scheduling/Assembly_Line_Scheduling.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+        tmp = []
+        for nThNode in range(nodes):
+            tmp.append(int(input("Node" + str(nThNode) + ": ")))
+        line.append(tmp)
+    
+    print("\n-- Transaction Time --")
+    for lineN in range(ASSEMBLY_LINES):
+        tmp = []
+        for nThNode in range(nodes):
+            if (nThNode==0):
```
Comment (MastersAbh) [inline Assembly_Line_Scheduling/Assembly_Line_Scheduling.py]: space around ==

**Comment by Karna98** [comment]
Updated changes.

**Pull request merged**